### DISCUSSION TITLE
how disable default css in head

### DISCUSSION THREAD

**Discussion by borodadada** [nuxt/nuxt#17439] [Questions] [answered]
Hi! How i can disable this styles? its not my.
![Snipaste_2022-06-16_10-17-30](https://user-images.githubusercontent.com/5166757/174014664-144f452e-707b-4d98-8087-dfb63fb9ec38.png)

**Comment by JulianMar** [comment, marked as answer]
Hey, if you build for production they are gone.
They might be some styles for a build indicator or something like that

**Reply by borodadada** [reply]
thx!
